source/installation/file-location.txt

```diff
@@ -0,0 +1,80 @@
+.. _file-location:
+
+==================================
+Relational Migrator File Locations
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page provides the location of files needed when working with 
+Relational Migrator. The location of the files varies according to the OS 
+you are working with. 
+
+.. note::
+
+    This information is correct for Relational Migrator v1.0.35 and higher (earlier 
+    versions store files in different locations).
+
+Mac
+---
+
+The file locations for the Mac OS:
+
+- :guilabel:`Configuration file`
+  
+  ``~/Library/Application Support/MongoDB/Relational Migrator/user.properties``
+
+- :guilabel:`Driver downloads`
+
+  ``/Library/Application Support/MongoDB/Relational Migrator/Drivers``
```

Comment:
(question)
Is a ~ part of the file path here as well?

Reply:
Yes, thank you!